CantEngine/Components/UIComponent.cpp

```diff
@@ -20,13 +25,25 @@ RTTR_REGISTRATION
 		.property("FinalRotation", &UIComponent::finalRotationValue)
 		.property("RotationRate", &UIComponent::rotationRate)
 		.property("Enabled", &UIComponent::m_enabled)
+		.property("TextInternalUI", &UIComponent::m_internalTextUI)
```

Comment:
Change TextInternalUI to Text. The name makes it less clear that it's normal text.
Also change TextFontType to TextFont. 
Might need to change the jsons too

Reply:
Do this for both the cpp and jsons and LUA.